_data/projects/dotnet-architecture.yml

```diff
@@ -0,0 +1,13 @@
+name: dotnet-architecture
+desc: Code example that follows the Microsoft Architecture guidance.
+site: https://github.com/dotnet-architecture/eShopOnWeb
+tags:
+  - web
+  - dotnet
+  - razor-pages
+  - .net
+  - docker
+  - ef-core
+upforgrabs:
+  name: help-wanted
```

Comment:
@mohjache A littile bit mistake here, your label is "help wanted" but name is "help-wanted". Please correct it.

Reply:
Thankyou for the quick feedback. I'lll get on this.